src/components/MenuBar/ButtonBar.js

```diff
@@ -69,7 +69,7 @@ function ButtonBar({ isMobile }) {
 		},
 		{
 			name: 'Gallery',
-			to: '/gallery'
+			to: '/gallery/hoth-8'
```

Comment:
I'm wondering if there's a better way than hardcoding this. I don't have ideas at the moment and if we don't think of any this is okay. Just make a comment somewhere that whenever we add new HOTH data we have to update this as well.